config.go

```diff
@@ -468,12 +468,9 @@ func loadConfig() (*config, []string, error) {
 		os.Exit(0)
 	}
 
-	// Exit if you tried to do rollback testing on a network other than
-	// simnet.
-	if cfg.RollbackTest && !cfg.SimNet {
-		fmt.Fprintln(os.Stderr, "Tried to do rollback testing of "+
-			"wallet for network other than simnet!")
-		os.Exit(0)
+	// Warn if rollback testing is enabled, as this feature was removed.
```

Comment:
Pretty sure I was in favor of erroring on old config in the past too.  I prefer actually forcing people to clean up config files but since that isn't how we do it, I'm fine with it.